packages/elements/src/components/ui/buttons/common/ButtonContent.module.css

```
@@ -1,12 +1,6 @@
 .iconLeft,
 .iconRight {
   color: var(--current-icon-color);
```

I would undo this and use `--current-icon-height` to set the height instead of setting it through the `size` prop. In this way, we can adapt the height to the button size, so the icon is smaller in a small button.

```suggestion
  color: var(--current-icon-color);
  height: var(--current-line-height);
}

.iconLeft,
.iconRight {
  height: var(--current-icon-height);
```

Go to `Button.module.css` and add these parts:
```
--swui-button-icon-height-small: 1.2rem;

&.small {
  --current-icon-height: var(--swui-button-icon-height-small);
}
```

You might also want to check with the designers if we should remove these while at it. They have never worked before but might start working if we make this change. Is that something we want?

```
--swui-button-icon-height-large: 2.4rem;

&.large {
  &.iconButton {
    --current-icon-height: var(--swui-button-icon-height-large);
  }
}
```